sharepoint/sharepoint-ps/sharepoint-online/Set-SPOCopilotPromoOptInStatus.md

```diff
@@ -34,8 +34,6 @@ Set-SPOCopilotPromoOptInStatus -IsCopilotPromoStatusEnabled $true
 
 Example 1 sets the Opt-In Copilot promo status for the tenant to `True`.
 
-### Parameter:
-
```

Comment:
this should be

```
## PARAMETERS
```

Reply:
@arakeshmicrosoft as you do have a param in your cmdlet, you do need the `## PARAMETERS` section. If you look at your [PR preview documentation](https://review.learn.microsoft.com/en-us/powershell/module/sharepoint-online/set-spocopilotpromooptinstatus?view=sharepoint-ps&branch=pr-en-us-765), you'll this is currently not properly formatted due to that:

![image](https://github.com/user-attachments/assets/1acbcb5b-bd3e-4296-925e-f9404856cc34)

It needs to be something like this (I just picked a [random file](https://raw.githubusercontent.com/MicrosoftDocs/OfficeDocs-Sharepoint-PowerShell/refs/heads/main/sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOSiteVersionPolicyJob.md)):

![image](https://github.com/user-attachments/assets/26cc6a08-1507-4665-b93f-c7b5f279903f)